pulsar-common/src/main/java/org/apache/pulsar/common/util/netty/EventLoopUtil.java

```diff
@@ -65,8 +65,7 @@ public static EventLoopGroup newEventLoopGroup(int nThreads, boolean enableBusyW
                     try {
                         CpuAffinity.acquireCore();
                     } catch (Throwable t) {
-                        log.warn("Failed to acquire CPU core for thread {}", Thread.currentThread().getName(),
-                                t.getMessage(), t);
+                        log.warn("Failed to acquire CPU core for thread {}",Thread.currentThread().getName(), t);
```

Comment:
Missing one space in ",Thread":
`log.warn("Failed to acquire CPU core for thread {}", Thread.currentThread().getName(), t);`

Reply:
I will fix it.